de.marw.cdt.cmake.core/src/main/java/de/marw/cdt/cmake/core/internal/BuildscriptGenerator.java

```diff
@@ -29,6 +33,13 @@
 import org.eclipse.cdt.core.settings.model.ICConfigurationDescription;
 import org.eclipse.cdt.core.settings.model.ICSourceEntry;
 import org.eclipse.cdt.core.settings.model.util.CDataUtil;
+import org.eclipse.cdt.make.core.IMakeTarget;
+import org.eclipse.cdt.make.core.IMakeTargetManager;
+import org.eclipse.cdt.make.core.MakeCorePlugin;
+import org.eclipse.cdt.make.core.makefile.IMakefileReaderProvider;
+import org.eclipse.cdt.make.core.makefile.ITargetRule;
+import org.eclipse.cdt.make.internal.core.makefile.MakefileReader;
```

Comment:
Please do not rely on internal packages.

Reply:
These commits were reverted, no internal dependencies are used now, sorry for the confusion. I will create two separate new pull requests.